config/checkstyle_input_suppressions.xml

```diff
@@ -524,6 +524,9 @@
   <suppress checks="Regexp" files="indentation[\\/]indentation[\\/]InputIndentationLambda3.java"/>
   <suppress checks="Regexp" files="indentation[\\/]indentation[\\/]InputIndentationLambda4.java"/>
   <suppress checks="Regexp" files="indentation[\\/]indentation[\\/]InputIndentationLambda5.java"/>
+  <suppress checks="Regexp" files="indentation[\\/]indentation[\\/]InputIndentationLambda6.java"/>
```

Comment:
please remove this suppressions and please place comments with config in Input files.

Reply:
Yes I had already placed config with the required format in all the Input files. 
Actually the comment rule (in indentation every line should end with comment of exp and actual) and the new rule for all input files(every file should end with `//ok` or `//violation`) is conflicting with each other. That's why I have to push it under suppression file. I think it need to be fixed.
See: https://github.com/checkstyle/checkstyle/issues/8725

Reply:
ok, got it, very confusing - our bad.

I reopened https://github.com/checkstyle/checkstyle/issues/8725
please do me a favor and send separate PR to fix it.